### PR TITLE
Add SEO information to archives

### PR DESCRIPTION
**Pull request by core23**
I am targeting this branch, because this is BC if not tag is created before merging this.

## Changelog

```markdown
### Added
- Added SEO information to archives
```

## To do
- [x] Update translation

## Subject

Adds more SEO information to archive pafges.

### CODE REVIEW

File: src/Action/CollectionPostArchiveAction.php [hunk 1]
```diff
@@ -51,6 +51,28 @@ public function __invoke(Request $request, $tag)
...
+        if ($seoPage = $this->getSeoPage()) {
+            $seoPage
+                ->setTitle($this->trans('archive_collection.meta_title', [
+                    '%title%' => $this->getBlog()->getTitle(),
+                    '%collection%' => $collection->getName(),
+                ], 'Core23NewsBundle'))
+                ->addMeta('property', 'og:title', $this->trans('archive_collection.meta_title', [
+                    '%title%' => $this->getBlog()->getTitle(),
+                    '%collection%' => $collection->getName(),
+                ], 'Core23NewsBundle'))
```
Comment (kunicmarko20) [inline src/Action/CollectionPostArchiveAction.php]: `Core23NewsBundle` 😄

File: src/Action/PostArchiveAction.php [hunk 2]
```diff
@@ -13,14 +13,38 @@
...
     /**
      * @return Response
      */
     public function __invoke(Request $request)
     {
+        if ($seoPage = $this->getSeoPage()) {
+            $seoPage
+                ->setTitle($this->trans('archive.meta_title', [
+                    '%title%' => $this->getBlog()->getTitle(),
+                ], 'Core23NewsBundle'))
```
Comment (kunicmarko20) [inline src/Action/PostArchiveAction.php]: same here

File: src/Action/TagPostArchiveAction.php [hunk 3]
```diff
@@ -48,6 +48,26 @@ public function __invoke(Request $request, $tag)
             throw new NotFoundHttpException('Unable to find the tag');
         }
 
+        if ($seoPage = $this->getSeoPage()) {
+            $seoPage
+                ->setTitle($this->trans('archive_tag.meta_title', [
+                    '%title%' => $this->getBlog()->getTitle(),
+                    '%tag%' => $tag->getName(),
+                ], 'Core23NewsBundle'))
```
Comment (kunicmarko20) [inline src/Action/TagPostArchiveAction.php]: one more

File: src/Action/CollectionPostArchiveAction.php (hunk 1 above)
Comment (core23) [reply src/Action/CollectionPostArchiveAction.php]: Still WIP ;)

**Comment by core23** [comment]
Please review again @greg0ire @kunicmarko20

**Comment by greg0ire** [comment]
Thanks @core23 !

**Pull request merged**